ts/src/binance.ts

```diff
@@ -4499,8 +4498,13 @@ export default class binance extends Exchange {
             }
             request['price'] = this.priceToPrecision (symbol, price);
         }
+        const timeInForce = this.safeString (params, 'timeInForce');
```

Comment:
@olegtaranenko You sure this is needed? At the end of the method we extend the `request` with the `params` 

```
 this.extend (request, requestParams);
```
so the timeInForce inside params has priority unless it's 'PO'

Reply:
yes, I'm sure. The evil is the code
```javascript
        if (timeInForceIsRequired) {
            request['timeInForce'] = this.options['defaultTimeInForce']; // 'GTC' = Good To Cancel (default), 'IOC' = Immediate Or Cancel
        }
```
which **always** overwrite my desired timeInForce. At the current state, I need to change exchange.options.defaultTimInForce property to bypass it. 

```javascript
const tif=exchange.options.defaultTimInForce
exchange.options.defaultTimInForce = 'GTX'
exchange.createOrder(....)
exchange.options.defaultTimInForce = tif
```

I don't think this is a good solution :-)

Reply:
ah... maybe you right... let me check again